File: LeetCode/C#/1290. Convert Binary Number in a Linked List to Integer.cs
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     public int val;
 *     public ListNode next;
 *     public ListNode(int val=0, ListNode next=null) {
 *         this.val = val;
 *         this.next = next;
 *     }
 * }
 */
public class Solution {
    public int GetDecimalValue(ListNode head) {
        var current = head;
        var bitstring = string.Empty;
        
        while(current != null)
        {
            bitstring += current.val;
            current = current.next;
        }
        
        return Convert.ToInt32(bitstring, 2);
    }
}

File: LeetCode/Java/1290. Convert Binary Number in a Linked List to Integer.java
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     int val;
 *     ListNode next;
 *     ListNode() {}
 *     ListNode(int val) { this.val = val; }
 *     ListNode(int val, ListNode next) { this.val = val; this.next = next; }
 * }
 */
class Solution {
    public int getDecimalValue(ListNode head) {
        String binaryString = "";
        int output = 0;

        while(head != null) {
            binaryString += head.val;
            head = head.next;
        }

        output = Integer.parseInt(binaryString, 2);
        return output;
    }
}